src/core/support/slice_buffer.c

```diff
@@ -117,7 +117,7 @@ void gpr_slice_buffer_add(gpr_slice_buffer *sb, gpr_slice s) {
                s.data.inlined.bytes, s.data.inlined.length);
         back->data.inlined.length += s.data.inlined.length;
       } else {
-        size_t cp1 = GPR_SLICE_INLINED_SIZE - back->data.inlined.length;
+		gpr_uint32 cp1 = GPR_SLICE_INLINED_SIZE - back->data.inlined.length;
```

Comment:
there are type casting problems on Win64 when using both size_t and 32bit int on am expression.
I think that cp1 will not bigger than INT_MAX. So, I changed it.
If there are something better solutions in this case, could you let me know?